Sources/gg.datagram.web-requests.sdPlugin/propertyinspector/http.html

```diff
@@ -30,12 +30,23 @@
       </select>
     </div>
 
-    <!--div type="textarea" class="sdpi-item" id="headers">
-      <div class="sdpi-item-label">Headers</div>
+    <div class="sdpi-item">
+      <div class="sdpi-item-label" data-localize="ContentType">Content Type</div>
+      <select class="sdpi-item-value select" id="contentType" required>
+        <option value="" disabled selected hidden></option>
```

Comment:
I think I would rather leave the placeholder option selectable. If you want to select a content type that's not in this list, it seems weird to have to pick a value anyway.